.dockerignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Let's try, since we "mount" our current directory into `/app`, we could avoid this copy